docker/dev/find_fastest_apt_source.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Define a list of mirrors without using Bash arrays.
+# c.f. https://stackoverflow.com/a/23930212/724872
+read -r -d '' APT_SOURCES << EOM
+mirrors.aliyun.com
+mirrors.ustc.edu.cn
+mirrors.163.com
+archive.ubuntu.com
+ftp.de.debian.org
+packages.debian.org
```

Comment:
`ftp.de.debian.or` and `packages.debian.org` are not ubuntu mirrors

Reply:
What should i do? Remove them?

Reply:
YES. maybe change them to other mirrors